src/main/java/org/jenkins/plugins/lockableresources/LockableResource.java

```diff
@@ -157,6 +157,25 @@ public String getLabels() {
     return labels;
   }
 
+  /**
+   * Get labels of this resource
+   * @return List of assigned labels.
+   */
+  @Exported
+  public List<String> getLabelsAsList() {
+    return Arrays.asList(labels.split("\\s+"));
+  }
+
+  /**
+   * Checks if the resource has label *labelToFind*
+   * @param labelToFind Label to find.
+   * @return {@code true} if this resource contains the label.
+   */
+  @Exported
+  public boolean hasLabel(String labelToFind) {
```

Comment:
Is this effectively an alias with a more intuitive name? Or is there some further plan how they could differ?
Like was there a reason to keep `private labelsContain()` instead of just making it `public`? Just cleaner to have less surprise in the API evolution?

Reply:
Because hasLabel is self-explanatory. So no surprise nor api extension are planned now